static/app/views/replays/detail/domMutations/index.tsx

```diff
@@ -153,7 +155,7 @@ const StepConnector = styled('div')`
   position: absolute;
   height: 100%;
   top: 28px;
-  left: 31px;
+  left: 12px;
   border-right: 1px ${p => p.theme.border} dashed;
 `;
```

Comment:
This connector css is broken. The line disappears when I scroll down below the fold, i think because it's position absolute, it's relative to the top of the first row.

<img width="709" alt="Screen Shot 2022-08-18 at 9 26 12 AM" src="https://user-images.githubusercontent.com/187460/185446284-73a76f42-f04f-4d26-9c26-054af3ab45de.png">